e2e/autoscript-apps/src/authz-tree-basic-redirect/autoscript.ts

```diff
@@ -107,7 +104,7 @@ function autoscript() {
           if (response.ok) {
             console.log('Request to IG resource successfully responded');
           } else {
-            throw new Error('IG Transactional Authorization was not successful');
+            throw new Error('IG Tree Based Authorization was not successful');
```

Comment:
If you're scrolling through the changes from the top, this is the first change that you can see where I'm fixing something on the authz-tree-basic tests that was referencing **Transactional Authorization** instead of **Tree Based Authorization**. I'll refer to test fixes for transactional / tree-based authorization in changes further down.